demos/twitter/src/main/java/com/datatorrent/demos/twitter/RSSFeedExtractor.java

```
@@ -0,0 +1,160 @@
+        Node nNode = nList.item(index);
+        if (nNode.getNodeType() == Node.ELEMENT_NODE) {
+          Element eElement = (Element) nNode;
+          URLToConvert = eElement.getElementsByTagName("value").item(0)
+              .getTextContent();
+        }
+      }
+
+    } catch (ParserConfigurationException e) {
+      throw new RuntimeException("Parser Configuration Exception");
+
+    } catch (IOException e) {
+      throw new RuntimeException("Runtime Exception");
+    }
+
+    try {
+      rssURL = new URL(URLToConvert);
+    } catch (MalformedURLException e) {
+      throw new RuntimeException("Malformed URL Exception");
+    }
+  }
+
+  @Override
+  public void teardown()
+  {
+    // TODO Auto-generated method stub
+    try {
+      in.close();
+    } catch (IOException e) {
+      throw new RuntimeException("IO Exception");
+    }
+  }
+
+  @Override
+  public void emitTuples()
+  {
+    // TODO Auto-generated method stub
+    try {
+      in = new BufferedReader(new InputStreamReader(rssURL.openStream()));
+      String sourceCode = "";
+      String line;
+      while ((line = in.readLine()) != null) {
+        if (line.contains("<title>")) {
+          int firstIndex = line.indexOf("<title>");
+          String index = line.substring(firstIndex);
+          index = index.replace("<title>", "");
+          int lastIndex = index.indexOf("</title>");
+          index = index.substring(0, lastIndex);
+          sourceCode += index + "\n";
+        }
+        if (line.contains("<link>")) {
+          int firstIndex = line.indexOf("<link>");
+          String index = line.substring(firstIndex);
+          index = index.replace("<link>", "");
+          int lastIndex = index.indexOf("</link>");
+          index = index.substring(0, lastIndex);
+          sourceCode += index + "\n";
+        }
+      }
+      in.close();
+      output.emit(sourceCode);
+    } catch (MalformedURLException e) {
+      throw new RuntimeException("Malformed URL Exception");
```

Please don't mask real exception.